### PR TITLE
Web scraping of Amazon Kindle #317

### PR DESCRIPTION
**Pull request by Tisha6661**
## Description
This is to scrape book information from amazon kindle website like book title, book author, ratings, price, book url.

## Resolves: 317

image

### CODE REVIEW

File: src/scrape_up/kindle_bookstore/kindle.py [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+    all_books={
+                 'Book_Title': all_titles,
+                 'Author_Name':all_authors,
+                 'Stars': all_stars ,
+                 'Price': all_price ,
+                 'URL':all_urls}
+
+    #Creating a dataframe using pandas
+    dataframe = pd.DataFrame.from_dict(all_books, orient='index')
+    dataframe = dataframe.transpose()
```
Comment (nikhil25803) [inline src/scrape_up/kindle_bookstore/kindle.py:94]: Do not add pandas as a package, no need to use this. We are not required to send the response in the dataframe format. We have to send it as a JSON. 
Check other modules for reference, and how they are implemented. Check `GitHub` module for the reference

**Review by nikhil25803 — CHANGES_REQUESTED** [review]

File: src/scrape_up/kindle_bookstore/kindle.py (hunk 1 above)
Comment (Tisha6661) [reply src/scrape_up/kindle_bookstore/kindle.py:94]: Ohk, I will make the these updates.

**Comment by Tisha6661** [comment]
Here are the updates-
image

image

**Review by nikhil25803 — CHANGES_REQUESTED** [review]
Looks good, just made some minor changes by myself.

**Comment by nikhil25803** [comment]
Great work @Tisha6661 !!
Do not forget to
+ Share your contribution on socials(Twitter mainly), tag us and Clueless 🫂
+ Give this repository a star ✨
+ Follow Clueless Community on GitHub ✅

**Pull request merged**